inc/classes/class-settings.php

```diff
@@ -0,0 +1,229 @@
+<?php
+/**
+ * Settings class.
+ *
+ * @package wp-google-login
+ */
+
+namespace WP_Google_Login\Inc;
+
+use WP_Google_Login\Inc\Traits\Singleton;
+
+/**
+ * Class Settings
+ */
+class Settings {
+
+	use Singleton;
+
+	/**
+	 * Plugin constructor.
+	 */
+	protected function __construct() {
+		$this->setup_hooks();
+	}
+
+	/**
+	 * To setup actions/filters.
+	 *
+	 * @return void
+	 */
+	public function setup_hooks() {
+
+		/**
+		 * Actions
+		 */
+		add_action( 'admin_menu', [ $this, 'add_admin_menu' ] );
+		add_action( 'admin_init', [ $this, 'settings_init' ] );
+	}
+
+	/**
+	 * Add admin menu.
+	 *
+	 * @return void
+	 */
+	public function add_admin_menu() {
+		if ( current_user_can( 'manage_options' ) ) {
+			add_filter( 'plugin_action_links', [ $this, 'plugin_action_links' ], 10, 2 );
+		}
+		add_options_page( 'WP Google Login', 'WP Google Login', 'manage_options', 'wp-google-login', [
```

Comment:
We need to make these static strings translatable.

Reply:
Revised